packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardWithFilters.tsx

```diff
@@ -0,0 +1,177 @@
+import { Divider, Grid, styled } from '@mui/material';
+import React from 'react';
+
+import CloudPulseIcon from 'src/assets/icons/entityIcons/monitor.svg';
+import { CircleProgress } from 'src/components/CircleProgress';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { Paper } from 'src/components/Paper';
+import { Placeholder } from 'src/components/Placeholder/Placeholder';
+import { useCloudPulseDashboardByIdQuery } from 'src/queries/cloudpulse/dashboards';
+
+import { CloudPulseDashboardFilterBuilder } from '../shared/CloudPulseDashboardFilterBuilder';
+import { CloudPulseTimeRangeSelect } from '../shared/CloudPulseTimeRangeSelect';
+import { FILTER_CONFIG } from '../Utils/FilterConfig';
+import {
+    checkIfFilterBuilderNeeded,
+  checkMandatoryFiltersSelected,
+  getDashboardProperties,
+} from '../Utils/ReusableDashboardFilterUtils';
+import { CloudPulseDashboard } from './CloudPulseDashboard';
+
+import type { FilterValueType } from './CloudPulseDashboardLanding';
+import type { TimeDuration } from '@linode/api-v4';
+
+/**
+ * These properties are required for rendering this reusable component
+ */
+export interface CloudPulseDashboardWithFiltersProp {
+  /**
+   * The id of the dashboard that needs to be rendered
+   */
+  dashboardId: number;
+  /**
+   * The resource id for which the metrics will be listed
+   */
+  resource: number;
+}
+
+export const CloudPulseDashboardWithFilters = React.memo(
+  (props: CloudPulseDashboardWithFiltersProp) => {
+    const { dashboardId, resource } = props;
+    const { data: dashboard, isError } = useCloudPulseDashboardByIdQuery(
+      dashboardId
+    );
+
+    const [filterValue, setFilterValue] = React.useState<{
+      [key: string]: FilterValueType;
+    }>({});
+
+    const [timeDuration, setTimeDuration] = React.useState<TimeDuration>({
+      unit: 'min',
+      value: 30,
+    });
+
+    const StyledPlaceholder = styled(Placeholder, {
+        label: 'StyledPlaceholder',
+    })({
+        flex: 'auto',
+    });
+
+    const onFilterChange = React.useCallback(
+      (filterKey: string, value: FilterValueType) => {
+        setFilterValue((prev) => ({ ...prev, [filterKey]: value }));
+      },
+      []
+    );
+
+    const handleTimeRangeChange = React.useCallback(
+      (timeDuration: TimeDuration) => {
+        setTimeDuration(timeDuration);
+      },
+      []
+    );
+
+    const renderPlaceHolder = React.useCallback((subtitle: string) => {
+        return (
+            <Paper>
+                <StyledPlaceholder
+                    icon={CloudPulseIcon}
+                    isEntity
+                    subtitle={subtitle}
+                    title=""
+                />
+            </Paper>
+        );
+    }, []);
```

Comment:
```suggestion
    const renderPlaceHolder = (subtitle: string) => {
        return (
            <Paper>
                <StyledPlaceholder
                    icon={CloudPulseIcon}
                    isEntity
                    subtitle={subtitle}
                    title=""
                />
            </Paper>
        );
    };
```
we can remove callback from here as this function is not going to be passed as props to other component